cogs/admin.py

```
@@ -0,0 +1,205 @@
+            description="Bot is shutting down...",
+            color=self.bot.config.WARNING_COLOR
+        )
+        await ctx.send(embed=embed)
+        await self.bot.close()
+    
+    @commands.command(
+        name="eval",
+        description="Evaluate Python code (dangerous - owner only)",
+        usage="eval <code>"
+    )
+    @is_owner()
```

Pylint (3.3.7)

[error] Too many positional arguments for method call

(E1121)